src/table/cells/handle-cell-focus.js

```diff
@@ -80,3 +80,49 @@ export const handleNavigateTop = ({ tableSection, focusedCellCoord, rootElement
     }
   }
 };
+
+const getMemoisedSrNotation = () => {
+  let prevSelectedCount = 0;
+
+  return ({ focusedCellCoord, rootElement, selState }) => {
+    if (focusedCellCoord[0] === 0) return '';
+
+    const [rowIdx, colIdx] = focusedCellCoord;
+    const rowElements = rootElement.getElementsByClassName('sn-table-row');
+    const cell = rowElements[rowIdx]?.getElementsByClassName('sn-table-cell')[colIdx];
+
+    const tCellContent = cell && cell.innerText;
+    const isCellSelected = cell && cell.classList.contains('selected');
+
+    let notation = '';
+
+    if (selState.rows.length) {
+      const isSingularSelection = selState.rows.length === 1;
+      const selectionNote = `There ${isSingularSelection ? 'is' : 'are'} ${selState.rows.length} selected value${
+        isSingularSelection ? '' : 's'
+      } currently.`;
+
+      if (prevSelectedCount < selState.rows.length) {
+        // if we select cell
+        notation += `Value is selected. ${selectionNote}`;
+      } else if (prevSelectedCount > selState.rows.length) {
+        // if we deselect cell
+        notation += `Value is deselected. ${selectionNote}`;
+      } else if (isCellSelected) {
+        // if we are in selection mode and move to selected cell
+        notation += `${tCellContent},value is selected.`;
+      } else {
+        // if we are in selection mode and move to unselected cell
+        notation += `${tCellContent},value is not selected.`;
+      }
+    } else if (selState.rows.length === 0 && prevSelectedCount > 0) {
+      // if we deselect last (selected) cell which means we close the selection mode
+      notation += 'Value deselected and exited selection mode.';
```

Comment:
We actually might want something like this for pressing enter and escape. I'd say that is bonus or at least a different PR

Reply:
yes, should we create a separate issue for it?